app/services/csv_import_service.rb

```diff
@@ -0,0 +1,18 @@
+class CsvImportService
+    require 'csv'
+  
+    def call(file)
+      opened_file = File.open(file)
+      options = { headers: true, col_sep: ';' }
+      CSV.foreach(opened_file, **options) do |row|
+  
+        # map the CSV columns to your database columns
+        user_hash = {}
+        user_hash[:keyword] = row['Identifier']
+  
+        SearchTerm.find_or_create_by!(user_hash)
+        # for performance, you could create a separate job to import each user
+        # CsvImportJob.perform_later(user_hash)
+      end
+    end
+  end
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/1d081ecd36c48dd47f0123e7164aa066b7a4d685/app/services/csv_import_service.rb#L18"></span><code>end</code> at 18, 2 is not aligned with <code>class</code> at 1, 0.</td>
    </tr>
  </tbody>
</table>